src/main/java/domain/value/ManualCount.java

```diff
@@ -0,0 +1,10 @@
+package domain.value;
+
+public record ManualCount(int value) {
+    public ManualCount {
+        if (value < 0) throw new IllegalArgumentException("수동 개수는 음수일 수 없습니다.");
+    }
+    public void validateAgainstTotal(int total) {
+        if (value > total) throw new IllegalArgumentException("수동 개수가 전체를 초과했습니다.");
+    }
```

Comment:
```suggestion
public record ManualCount(int value) {

    public ManualCount {
        if (value < 0) throw new IllegalArgumentException("수동 개수는 음수일 수 없습니다.");
    }

    public void validateAgainstTotal(int total) {
        if (value > total) throw new IllegalArgumentException("수동 개수가 전체를 초과했습니다.");
    }
```

1. 메서드별로 최대한 줄바꿈 해주시면 좋아요~

2. 나중에 에러가 발생 했을 때 `수동 개수는 음수일 수 없습니다.`, `수동 개수가 전체를 초과했습니다`로 바로 파악하기엔 어려워서 에러 메시지 내용을 보강해야할 필요성이 보여요

Reply:
넵! 수정했습니다!